libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -449,6 +449,8 @@ constructor(
         fasterRouteController.stop()
     }
 
+    internal fun getNavigationSession() = navigationSession
```

Comment:
Could we expose `internal` `registerNavigationSessionStateObserver`, `unregisterNavigationSessionStateObserver` and `unregisterAllNavigationSessionStateObservers` APIs instead as we do for the rest of the observers. That's how we do currently in `MapboxNavigationTelemetry` too 👀 https://github.com/mapbox/mapbox-navigation-android/blob/e039b7db455ef2732701f52b47e568a6ebf52f83/libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt#L590-L608

Reply:
👍